src/main/java/no/nav/foreldrepenger/regler/uttak/fastsetteperiode/betingelser/aktkrav/SjekkOmMorOppgittUføre.java

```diff
@@ -0,0 +1,27 @@
+package no.nav.foreldrepenger.regler.uttak.fastsetteperiode.betingelser.aktkrav;
+
+import java.util.Objects;
+
+import no.nav.foreldrepenger.regler.uttak.fastsetteperiode.FastsettePeriodeGrunnlag;
+import no.nav.foreldrepenger.regler.uttak.fastsetteperiode.grunnlag.MorsAktivitet;
+import no.nav.fpsak.nare.doc.RuleDocumentation;
+import no.nav.fpsak.nare.evaluation.Evaluation;
+import no.nav.fpsak.nare.specification.LeafSpecification;
+
+@RuleDocumentation(SjekkOmMorOppgittUføre.ID)
+public class SjekkOmMorOppgittUføre extends LeafSpecification<FastsettePeriodeGrunnlag> {
+
+    public static final String ID = "AVSLAG_AKT_12";
+    public static final String BESKRIVELSE = "Har søker oppgitt at mor mottar uføretrygd ?";
+
+    public SjekkOmMorOppgittUføre() {
+        super(ID);
+    }
+
+    @Override
+    public Evaluation evaluate(FastsettePeriodeGrunnlag fastsettePeriodeGrunnlag) {
+        var morsAktivitetIPeriode = fastsettePeriodeGrunnlag.getAktuellPeriode().getMorsAktivitet();
+        return !fastsettePeriodeGrunnlag.isMorRett() && fastsettePeriodeGrunnlag.isFarRett() &&
```

Comment:
synes det blir litt rart å sjekke at bare far har rett her. Nok å sjekke om morsaktivitet er ufør?

Reply:
Husk man kan teoretisk havne inn i avslagsflyten for både Fellesperiode og Utsettelse 
Trigger var testcase mor_uten_bekreftet_uføretrygd_skal_til_manuell som hadde fellesperiode og uføre ...